CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.8.0 (Published as Beta)
```

Comment:
If we are fixing an issue and not adding anything new, this should be 1.7.1

Reply:
I was thinking only fixing a few, therefore the beta package; and then fix all icons in a formal release, possibly with new icons if Glenn ever writes back to us

Reply:
I would probably still just do it as a patch version since we are not adding any new features/functionality (just bug fixes). But you can def do them incrementally and make the first few available in beta.0